app/models/miq_queue.rb

```diff
@@ -19,6 +19,7 @@
 #
 class MiqQueue < ApplicationRecord
   belongs_to :handler, :polymorphic => true
+  has_many :binary_blobs, :as => :resource, :dependent => :nullify
```

Comment:
The dependent bit is going to add a query to every delete...we may just want to put nothing here.